### ISSUE TITLE
Changes to Text on Forms

### ISSUE THREAD

**Issue by WilhelminavanDijk** [ldbase/ldbase_issues#49]
I'm going to make this issue a single one for all the changes, but will upload doc files for each form separate.

I set it up to go through the forms from top to bottom, with the taxonomy name for the field (highlighted in grey), and then rows for the field name, help text, and examples. In a second column I indicated the actions that need to be taken for each row. E.g., No Change, Change text, move (help text) to descriptor spot, etc.

I hope it will be clear but please comment here or flock me if it doesn't work!

**Comment by WilhelminavanDijk** [comment]
Here is the first file, for the "add/edit project" form

[Change_Project.docx](https://github.com/ldbase/ldbase_issues/files/5272780/Change_Project.docx)

**Comment by barsenault-fsu** [comment]
@WilhelminavanDijk what does "Move text to descriptor spot" mean?  

In the Change_Project document, for "Description - Help Text" you put "move text to descriptor spot" and then "change descriptor to: ..." so I read that as change the descriptor text to the given text, make the description text always visible (not under question mark), and there will be no help tooltip.

But in the same document for "Project Investigator(s) - Help Text" you again put "Move text to descriptor spot" but you also want both Descriptor and Help text changed.  I asked myself, "Why move text if it is changing?" and now I'm not sure I understand what you mean by "move text to descriptor spot" if you are still going to have Help and Descriptor text.

From my point of view, "Help Text" is what appears when a user hovers over the question mark icon next to the field label or name.  The "Descriptor" text is the visible text like currently for Project Organizations, which reads "Please enter the unabbreviated name of each organization."  

Are you asking to get rid of all the tooltips?

**Comment by WilhelminavanDijk** [comment]
Yes! We want most of the text that is now under the question mark to be changed to always visible. and that is what i meant with move to descriptor spot. I can change this language in my next form ;) I can make an extra row for descriptor, for example.

We think that in many cases we're asking users to input pretty specific things, but we're not sure if people will actually hover and read, and then may not enter fields the way we want them to.  In a small number of instances, we have additional text that we intend to go under questions mark (I named that help text), but if you think that will be confusing, we can discuss if all text should become descriptor, and we don't have anymore "help" text. (we were thinking it might get too wordy in some cases).

**Comment by barsenault-fsu** [comment]
Yes if you have a row each for Help Text and Descriptor Text, that would be easy to read.  Help text will appear in the hover tooltip and Descriptor Text will appear above the form field.  If you don't provide Help Text for a field, then a tooltip question mark icon will not appear for that field.

So I will ignore the "Move text to descriptor spot" and just put the DESCRIPTOR and HELP text that you provided in the Change_Project document.  If you only provided Descriptor text, then I will remove the Help text currently there.

**Comment by barsenault-fsu** [comment]
In the Change_Project document, for the fields
- Project Descriptor(s)
- Educational Environment
- Educational Curricula
- Developmental Design

you asked for the Example Text to "Prepopulate the first term".  If we do that, then the user will have to manually clear that term in each field or it will get saved as part of their entry.

Might I suggest that we have gray placeholder text that reads "Click here to select or enter items" or something similar instead?
(edited to remove request to Favenzio.  I was able to change the labels on the display)

**Comment by WilhelminavanDijk** [comment]
That seems fine, as long as there is something? Right now those are just blank :).

**Comment by WilhelminavanDijk** [comment]
[Change_DataSet.docx](https://github.com/ldbase/ldbase_issues/files/5293060/Change_DataSet.docx)

Here is the docu for the Data-set form :)

**Comment by WilhelminavanDijk** [comment]
[Change_Code.docx](https://github.com/ldbase/ldbase_issues/files/5307503/Change_Code.docx)
[Change_Codebook.docx](https://github.com/ldbase/ldbase_issues/files/5307504/Change_Codebook.docx)
[Change_Document.docx](https://github.com/ldbase/ldbase_issues/files/5307505/Change_Document.docx)

Here are the code, codebook, and document text changes!

**Comment by barsenault-fsu** [comment]
@Favenzio There are some label changes in the Dataset form that will need changing in the Dataset view.  They are listed here as "old label" -> "becomes this label":

1. Dataset Contributors -> Dataset Contributor(s)
2. When was the data in this dataset collected? -> When were the data in this dataset collected?
3. Data Collection Locations -> Data Collection Location(s)
4. Assessment Name -> Assessment(s) Used
5. Participant Type -> Participants

Thank you!

**Comment by barsenault-fsu** [comment]
I have made changes to the Project, Dataset, Code, Codebook, and Document forms (and view labels when needed).

When Test is updated, I will label this issue as Needs Review.

**Comment by WilhelminavanDijk** [comment]
This looks good so far. I want to keep this issue open, because we may have text changes for the person page and possibly search page when they are finalized.

**Comment by Favenzio** [comment]
Solution has been accepted. Please create new issues if other changes are required.
